.github/workflows/ci.yml

```diff
@@ -0,0 +1,48 @@
+name: ci
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        node-version: [6.x, 8.x, 10.x, 12.x]
+        os: [ubuntu-16.04, windows-2016, macOS-10.14]
+
+    steps:
+      - uses: actions/checkout@v1
+      - name: Use Node.js ${{ matrix.node-version }} in ${{ matrix.os }}
```

Comment:
BTW, the system automatically appends the Node and OS version. So if this were `name: foo` then the tests would show up like `foo (ubuntu-16.04, node-10.x)`.

See https://github.com/ldapjs/node-ldapjs/runs/222797629#step:1:1